.github/.codecov.yml

```diff
@@ -12,6 +12,7 @@ coverage:
         target: auto
         threshold: 1%
         base: auto
+    patch: off # disable codecov/patch
```

Comment:
`codecov/patch` complains about the 

` /* eslint-disable @typescript-eslint/no-magic-numbers */` comments added in this PR. 

**Is there a better way to do it other than disable it ?**

<img width="957" alt="image" src="https://github.com/ckb-js/lumos/assets/7459812/db9dc95a-7e46-4562-9cf9-027f950c3328">

Reply:
this is because of some non-covered functions in the `mnemonic.ts` and this PR added some lines in `mnemonic.ts` that decreased the coverage rate. The report from Codecov is expected, so we should keep the patch coverage to understand the test coverage of a PR

Reply:
OK. Enabled `codecov/patch` .